src/navigator/SettingsMenu.tsx

```diff
@@ -0,0 +1,229 @@
+import { NativeStackScreenProps } from '@react-navigation/native-stack'
```

Comment:
Much of this file is just boilerplate, nearly identical to what we're doing in our current settings screen. The only real "new" stuff here is the Profile menu item.